### ISSUE TITLE
Edits for Hot Keys and Help Content

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#195]
@jessegreenberg, I'm creating a separate issue for the Hot Keys and Help document. There is a known content issue which I think it might be mentioned in another issue. I will review it today and post it here.

- [x] Taliesin to review current version.
- [x] Taliesin to reassign to @jessegreenberg after review.

**Comment by jessegreenberg** [comment]
Thanks @terracoda, please assign to me once you know what needs to be changed.

**Comment by terracoda** [comment]
@jessegreenberg, I just shared the following google document with you.
The only current incorrect information is the last heading.
"Hotkeys for jumping" should be "Grab Balloon"

Iterations on the actual content should probably wait until after interviews; however, I think I'd like to put the jump keys first. I will re-assign issue after I discuss with @emily-phet.

**Comment by jessegreenberg** [comment]
Thanks @terracoda, incorrect label should be fixed in commit above.  Please let me know if any other changes should be made.

**Comment by jessegreenberg** [comment]
Also, removing the high priority label since that was fixed, but feel free to add it again if you think it is necessary.

**Comment by terracoda** [comment]
@jessegreenberg, I'd like to go with the simple correction and also a re-order. Can you put the first section, "Keys for Dragging and Rubbing" at the bottom after the the "Grabbing Balloon" section.

No other changes for now.

**Comment by terracoda** [comment]
@jessegreenberg, FYI, the draft in the google doc is not relevant for now.

**Comment by jessegreenberg** [comment]
@terracoda like this?

![example](https://cloud.githubusercontent.com/assets/6396244/20078352/6ff13a7a-a4fd-11e6-81c7-4c8c9e167f9d.PNG)

**Comment by terracoda** [comment]
@jessegreenberg, order is good, but actually, if there is time, I'd prefer to update the content, too. I've updated the content in the google doc and highlighted the changes.

Sorry for the in decision on my part.

Please note the spaces in "Hot keys" and "Space bar" and the addition of of "key" after "Enter" and "Shift".

https://docs.google.com/document/d/1hb4y9r5X6RVgFD6OOQy8i3SlGYTCkskU8R2s7V3mB0Y/edit#

**Comment by jessegreenberg** [comment]
Sure, thanks for highlighting the changes, that made it really easy to see what needed to be changed.

How is this?
![example](https://cloud.githubusercontent.com/assets/6396244/20080364/97f7beb4-a506-11e6-8fa0-85ce362481ef.PNG)

**Comment by terracoda** [comment]
@jessegreenberg, sorry mistake in 2nd to last line:
, or Shift key plus S to drag __down__ a lot.

**Comment by jessegreenberg** [comment]
Sorry @terracoda, fixed in the commit above.  Are we ready to close this issue?

**Comment by terracoda** [comment]
@jessegreenberg, I missed the highlighting on "Hot Keys for Jumping" heading. We need a space there, too.

**Comment by terracoda** [comment]
And then you can close the issue, and thanks @jessegreenberg!

**Comment by jessegreenberg** [comment]
Fixed above, good catch @terracoda!
